docs/release/updating-to-osg-36.md

```diff
@@ -132,11 +146,33 @@ To update OSG-Configure, perform the following steps:
 
 #### HTCondor-CE ####
 
-AI (BrianL): https://opensciencegrid.atlassian.net/browse/SOFTWARE-4501
-HTCondor-CE in front of HTCondor CE, [see updating your HTCondor Hosts](#updating-your-htcondor-hosts)
+The OSG 3.6 release series contains [HTCondor-CE 5](https://htcondor.github.io/htcondor-ce/releases/#500), a major
+version upgrade from HTCondor-CE 4, which was available in the OSG 3.5 release repositories.
+To update HTCondor-CE, perform the following steps:
+
+1.  Merge any `*.rpmnew` files in `/etc/condor-ce/`
```

Comment:
"yum update if you haven't already" or something along those lines? Alternatively, say something like "Perform the following steps to migrate your configuration after updating HTCondor-CE"?

Reply:
Eh, maybe not.